cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -26,6 +26,9 @@ runtime-spec-compatible `dest/config.json`.
 **--type**
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
+**--log-level**
+  Log level to create runtime bundle. To be one of panic, fatal, error, warn, info, or debug. If unset, the level is identical to info.
```

Comment:
runtime-tools has a warning in-flight with opencontainers/runtime-tools#256.  I still think the default should be `error`.  I agree that the info messages you linked to are useful output, but I'd rather handle them by printing TAP to stdout instead of by using logrus to conditionally print them to stderr.  On the other hand, TAP is a big shift from where we are now; I'm fine handling that change in a follow-up PR.